plugins/GSdx/GSRenderer.cpp

```diff
@@ -221,22 +221,27 @@ bool GSRenderer::Merge(int field)
 		src[i] = GSVector4(r) * scale / GSVector4(tex[i]->GetSize()).xyxy();
 		src_hw[i] = (GSVector4(r) + GSVector4 (0, y_offset[i], 0, y_offset[i])) * scale / GSVector4(tex[i]->GetSize()).xyxy();
 
-		GSVector2 off(0, 0);
+		GSVector2 offset(0, 0);
+		GSVector2i diff(dr[i].left - baseline.x, dr[i].top - baseline.y);
 
 		// Time Crisis 2/3 uses two side by side images when in split screen mode.
-		off.x = tex[i]->GetScale().x * (dr[i].left - baseline.x);
+		// Though ignore cases where baseline and display rectangle offsets only differ by 1 pixel, causes blurring and wrong resolution output on FFXII
+	    if(diff.x >= 2)
+		{
+			offset.x = tex[i]->GetScale().x * diff.x;
+		}
 
-		if(dr[i].top - baseline.y >= 4) // 2?
+		if(diff.y >= 4) // Shouldn't this be 2?
```

Comment:
> Is the line too old to figure out when and why it was written?

Yeah, the last change which touched that part of merge circuit was at [2009](https://github.com/PCSX2-Alpha/pcsx2/commit/3b22adc4b350d0451f0fca99b75732650ff5afc1) !

> I'll make a private custom build for it and test it for a while. Even if it doesn't need changing, we should at least know why it's 4.

cool, thanks !